### ISSUE TITLE
samples

### ISSUE THREAD

**Issue by ppisljar** [anyvoiceai/Barkify#6]
could you provide some samples of the results ?

thank you

**Comment by anyvoiceai** [comment]
Yes. We do have some results and we would upload our model checkpoints this week.

**Comment by nickovchinnikov** [comment]
@anyvoiceai any updates?

**Comment by platform-kit** [comment]
@anyvoiceai Any movement on this repo? I'd love to see the results. If this repo works, that's a huge deal.
